storybook/.storybook/main.ts

```diff
@@ -24,13 +24,13 @@ const config: StorybookConfig = {
   },
   staticDirs: ['../public'],
   logLevel: 'debug',
-  async viteFinal(config) {
-    return mergeConfig(config, {
-      resolve: {
-        preserveSymlinks: true,
-      },
-    });
-  },
+  // async viteFinal(config) {
+  //   return mergeConfig(config, {
+  //     resolve: {
+  //       preserveSymlinks: true,
+  //     },
+  //   });
+  // },
```

Comment:
Do you need to use this to see your changes ? With just a `npm run watch` in `ui-core`, it works for me

Reply:
not sure, seems not to work sometimes with this enabled